src/index.html

```diff
@@ -7,7 +7,324 @@
+          </p>
+          <div class="description__buttons">
+            <a href="#" class="description__link">
+              Who am I
+            </a>
+            <a href="#" class="description__link">
+              What do i do
+            </a>
+          </div>
+        </div>
+      </section>
+    </header>
+
+    <main class="main">
+      <section class="main__section">
+        <div class="info">
+          <div class="info__description">
+            <h2 class="info__title">
+              Why Thrive?
+            </h2>
+            <p class="info__text">
+              <span class="info__text--bold">
+                Want to improve your well-being from the comfort of your own couch? Are you having trouble finding the right therapist?
+              </span>
+              Here at ThriveTalk, our licensed therapists provide the same quality care you would get in office from anywhere you can
+              access your laptop or mobile phone. Become your best self with ThriveTalk. Start therapy now with a licensed therapist!
+            </p>
+          </div>
+          <img
+            src="\images\info-section.png"
+            alt="Girl trains"
+            class="info__img"
+          >
+        </div>
+      </section>
+
+      <section class="main__section" id="about">
+        <div class="about">
+          <div class="about__info">
+            <h3 class="about__title">
+              About us
+            </h3>
+            <p class="about__text">
+              We want to help you thrive! Whether you are just looking for someone to talk to, or are struggling with a mental
+              wellness issue we’re here to help. Our highly talented therapists can help you with a range of issues including
+              relationships, sex, PTSD, depression, social anxiety, or even just caring for yourself more.
+            </p>
+          </div>
+          <h3 class="about__subtitle">
+            We can help you with
+          </h3>
+          <div class="about__card card">
+            <div class="card__type card__type--lifting">
+              <h2 class="card__title">
+                Weight Lifting
+              </h2>
+            </div>
+            <div class="card__type card__type--running">
+              <h2 class="card__title">
+                Running & Spinning
+              </h2>
+            </div>
+            <div class="card__type card__type--pumping">
+              <h2 class="card__title">
+                Pumping Iron
+              </h2>
+            </div>
+            <div class="card__type card__type--iron">
+              <h2 class="card__title">
+                Pumping Iron
+              </h2>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="main-section" id="services">
+        <div class="services">
```

Comment:
можливо ти мав на увазі -
```suggestion
      <section class="main__section" id="services">
        <div class="services">
```